hazelcast/src/main/java/com/hazelcast/internal/partition/impl/DefaultPartitionReplicaInterceptor.java

```diff
@@ -36,8 +36,6 @@ public void replicaChanged(int partitionId, int replicaIndex, PartitionReplica o
         if (replicaIndex == 0) {
             partitionService.getReplicaManager().cancelReplicaSync(partitionId);
         }
-        // RU_COMPAT_4_0 todo: where is partition version incremented? do we really need to update stamp?
-        //                previously this was also incrementing partition state version which is gone now
```

Comment:
I don't fully understand this change. It should be ok just didn't get it.

Reply:
there is no pending todo here... the comment was introduced in a previous release, when partition state version was replaced with partition state stamp